packages/fxa-payments-server/src/App.tsx

```diff
@@ -27,6 +28,7 @@ type AppProps = {
   store: Store,
   queryParams: QueryParams,
   navigateToUrl: (url: string) => void,
+  getScreenInfo: () => ScreenInfo,
```

Comment:
Why do these return the `ScreenInfo` singleton (or class?), but in `MockApp.tsx` it instantiates a new one?

Reply:
That line is actually a function type definition - that is, `getScreenInfo` should be a function that returns an instance of `ScreenInfo`